workers_launch_template.tf

```diff
@@ -530,6 +530,11 @@ resource "aws_launch_template" "workers_launch_template" {
         )}-eks_asg"
       },
       var.tags,
+      {
+        for tag in lookup(var.worker_groups_launch_template[count.index], "tags", local.workers_group_defaults["tags"]) :
+        tag["key"] => tag["value"]
+        if tag["key"] != "Name" && tag["propagate_at_launch"]
+      }
```

Comment:
Can you please follow the `instance` tag specifications tag extension ? I would be nice to have the same behavior across all tags in launch template.

Reply:
Thanks a lot for such a quick response.
Unfortunately, there is a bit different approach for the instance tagging is already defined with this change [#1092](https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1092) and this improvement [#1095](https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1095)
As a result, EC2 instances can get additional tags via the `propagate_at_launch` option in ASG. If I rewrite this in the same way as for volumes tagging via `tag_specifications` we'll lose the feature to tag instances created with launch configuration. Is it ok for us? If yes, I'm happy to do this.
Anyway, I can add this tags extension to the instances tag specification and it'll override the approach in ASG tagging, but it could lead to ambiguity in the future.
Please, share your thought regarding this?

Reply:
@barryib, do you still want me to make a change for the instances tagging with overriding the existing approach?

Reply:
Hi, @barryib . Sorry, maybe I misunderstood you. Do you mean that I should implement tagging of the volumes in the same way as tagging of the instances is already done?
Or should I modify the existing approach of the instances tagging in the same way as I've proposed to do for volumes?

We've really interested in the delivery of this feature, so could you advise, please, what should be done to get it merged?